var/spack/repos/builtin/packages/omega-h/package.py

```diff
@@ -18,6 +18,7 @@ class OmegaH(CMakePackage):
     maintainers = ['cwsmith']
     tags = ['e4s']
     version('main', branch='main')
+    version('10.1.0', commit='e88912368e101d940f006019585701a704295ab0',  git="https://github.com/SCOREC/omega_h.git")
```

Comment:
If it's a fork, unless you've worked with the original maintainers to take ownership of it, annotate the version so it's unambiguously not the mainline version, should they ever release a 10.1.0, so either:
```suggestion
    version('scorec.10.1.0', commit='e88912368e101d940f006019585701a704295ab0',  git="https://github.com/SCOREC/omega_h.git")
```
or 
```suggestion
    version('10.1.0.scorec', commit='e88912368e101d940f006019585701a704295ab0',  git="https://github.com/SCOREC/omega_h.git")
```
(and in the latter case, add `preferred=True` to the mainline 9.34.1 so that the forked version isnt' the default one.)